recipients_app/lib/ui/icons/payment_status_icon.dart

```diff
@@ -14,50 +13,14 @@ class PaymentStatusIcon extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    late Color backgroundColor;
-    late Widget? child;
-
-    switch (status) {
-      case PaymentUiStatus.confirmed:
-        backgroundColor = AppColors.primaryColor;
-        child = const Icon(
-          size: 14,
-          Icons.check_rounded,
-          color: Colors.white,
-        );
-      case PaymentUiStatus.empty:
-      case PaymentUiStatus.toBePaid:
-        backgroundColor = AppColors.backgroundColor;
-        child = null;
```

Comment:
The states "PaymentUiStatus.empty" and "PaymentUiStatus.toBePaid" are different than before. Is that intended and ok?

Reply:
I'm honestly not really sure because they were different in two different places as well. If you know which one the correct one is please feel free to adjust 😊🙏

Reply:
Do you know where I can see "PaymentStatusIconWithText" in the UI? I only can see "PaymentStatusIcon" but do not get where I can see "PaymentStatusIconWithText"???

Reply:
I found it :)

Reply:
PaymentStatusIcon are shown on this page:
![Screenshot_20250203-202124](https://github.com/user-attachments/assets/e1b54954-bdc1-4d93-9f44-55686204f627)

PaymentStatusIconWithTexts are shown on this Payments page:
![Screenshot_20250203-211442](https://github.com/user-attachments/assets/dc696619-4e90-46a5-884c-495f6c34cf12)

As far as I can see, the payments page never shows PaymentUiStatus.empty or PaymentUiStatus.toBePaid payments. So I would set the "PaymentUiStatus.color" to "AppColors.backgroundColor" and set PaymentUiStatus.color also to "AppColors.backgroundColor", so that the icon is not shown.

Reply:
See also my commit "Adjust colors to avoid ui changes for "PaymentStatusIcon""

Reply:
Do we have to adjust the golden tests after I changed the colors a bit??
btw. how do I execute the golden tests?